kasa/smartbulb.py

```diff
@@ -199,8 +201,8 @@ class SmartBulb(SmartDevice):
     SET_LIGHT_METHOD = "transition_light_state"
     emeter_type = "smartlife.iot.common.emeter"
 
-    def __init__(self, host: str) -> None:
-        super().__init__(host=host)
+    def __init__(self, host: str, *, port=TPLinkSmartHomeProtocol.DEFAULT_PORT) -> None:
```

Comment:
See my comment about defaulting the port parameter as none. I think we may need rewire the constructors to take a protocol instance to use (see my note about the authenticated kasa protocol above), and if so, not depending on the TPLinkSmartHomeProtocol class consts will make it simpler to pull off.

Reply:
I updated PR.